cbmc/patches/0001-Make-_IotMqtt_SerializePingreq-nondet-for-IotMqtt_Connect.patch

```diff
@@ -0,0 +1,29 @@
+From 9461f210f9a5d803064c39cf93c22ad0d5087538 Mon Sep 17 00:00:00 2001
+From: "Felipe R. Monteiro" <felisous@amazon.com>
+Date: Tue, 14 Apr 2020 00:09:28 +0000
+Subject: [PATCH] Make _IotMqtt_SerializePingreq nondet for IotMqtt_Connect
+ coverage
+
+Signed-off-by: Felipe R. Monteiro <felisous@amazon.com>
+---
+ libraries/standard/mqtt/src/iot_mqtt_serialize.c | 4 +++-
+ 1 file changed, 3 insertions(+), 1 deletion(-)
+
+diff --git a/libraries/standard/mqtt/src/iot_mqtt_serialize.c b/libraries/standard/mqtt/src/iot_mqtt_serialize.c
+index 016111e..2e92bc5 100644
+--- a/libraries/standard/mqtt/src/iot_mqtt_serialize.c
++++ b/libraries/standard/mqtt/src/iot_mqtt_serialize.c
+@@ -955,7 +955,9 @@ IotMqttError_t _IotMqtt_SerializePingreq( uint8_t ** pPingreqPacket,
+     /* Print out the PINGREQ packet for debugging purposes. */
+     IotLog_PrintBuffer( "MQTT PINGREQ packet:", pPingreq, MQTT_PACKET_PINGREQ_SIZE );
+ 
+-    return IOT_MQTT_SUCCESS;
```

Comment:
You want to surround this with an ifdef CBMC_SerializePingreq_NONDET or something